### DISCUSSION TITLE
Protecting Branch

### DISCUSSION THREAD

**Discussion by igbanam** [RubyConf-NG/openemrs#30] [Ideas] [answered]
Can we protect the `develop` branch with a "requires pull request". This way, those actively developing can get notified when the codebase is about to change

**Comment by robocopkaka** [comment, marked as answer]
Makes sense. I'd do that later today

**Reply by robocopkaka** [reply]
Done this. Develop would require an approval to merge but we can both bypass it

**Reply by igbanam** [reply]
Thank you. I'll try to ensure that there's no need for us to bypass it 🤞🏾
